redshift/redshift.go

```diff
@@ -79,19 +73,22 @@ func (r *Redshift) CopyGzipCsvDataFromS3(schema, table, file, awsRegion string,
 	}
 	copyCmd := fmt.Sprintf(
 		`COPY "%s"."%s" (%s) FROM '%s' WITH REGION '%s' GZIP CSV DELIMITER '%c'`,
-		schema, table, strings.Join(cols, ", "), file, awsRegion, delimiter)
-	copyCmd += " IGNOREHEADER 0 ACCEPTINVCHARS TRUNCATECOLUMNS TRIMBLANKS BLANKSASNULL EMPTYASNULL DATEFORMAT 'auto' ACCEPTANYDATE COMPUPDATE ON"
-	_, err := r.logAndExec(copyCmd, true)
+		schema, table, strings.Join(cols, ", "), file, r.s3Info.Region, delimiter)
+	opts := " IGNOREHEADER 0 ACCEPTINVCHARS TRUNCATECOLUMNS TRIMBLANKS BLANKSASNULL EMPTYASNULL DATEFORMAT 'auto' ACCEPTANYDATE COMPUPDATE ON"
+	creds := fmt.Sprintf(` CREDENTIALS 'aws_access_key_id=%s;aws_secret_access_key=%s'`, r.s3Info.AccessID, r.s3Info.SecretKey)
+	fullCopyCmd := fmt.Sprintf("%s%s%s", copyCmd, opts, creds)
+	_, err := r.logAndExec(fullCopyCmd)
```

Comment:
Nit: can we make this the same style as the CopyJSONDataFromS3 function?
Either
```
opts = <bar>
creds = <foo>
cmd = <qux>
%s%s%s cmd,opts,creds
```

or
```
creds = <foo>
copy <opts> %s creds
```

Rather than mixing both? It's a bit more confusing to read about.